org.eclipse.scout.rt.shared/src/main/java/org/eclipse/scout/rt/shared/http/async/AbstractAsyncHttpClientManager.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2010, 2023 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.scout.rt.shared.http.async;
+
+import java.io.IOException;
+
+import org.apache.hc.client5.http.impl.async.CloseableHttpAsyncClient;
+import org.apache.hc.client5.http.impl.async.HttpAsyncClients;
+import org.eclipse.scout.rt.platform.ApplicationScoped;
+import org.eclipse.scout.rt.platform.IPlatform.State;
+import org.eclipse.scout.rt.platform.IPlatformListener;
+import org.eclipse.scout.rt.platform.PlatformEvent;
+import org.eclipse.scout.rt.platform.exception.ProcessingException;
+
+/**
+ * <p>
+ * Acts as factory and cache for {@link org.apache.hc.client5.http.impl.async.CloseableHttpAsyncClient} (client is
+ * initialized only once); implementations use different {@link HttpAsyncClients} methods and configurations to build
+ * this client.
+ * </p>
+ * <p>
+ * Also acts a {@link IPlatformListener}, listening for platform change events. In case of a
+ * {@link State#PlatformStopping} event, the underlying {@link CloseableHttpAsyncClient} is shut down.
+ * </p>
+ */
+@ApplicationScoped
+public abstract class AbstractAsyncHttpClientManager<BUILDER> implements IPlatformListener {
+
+  /**
+   * Is {@link AbstractAsyncHttpClientManager} still active?
```

Comment:
Javadoc without ?: Suggestion: Flag indicating if {@link ...} is still active.

Reply:
Fixed in https://github.com/eclipse-scout/scout.rt/compare/f48c418ea822284b166f9970e4124a60e383bd63..e779671a1c3dd3fdd03df96227b29f95f566207b